ibm/provider.go

```diff
@@ -436,12 +437,13 @@ func Provider() *schema.Provider {
 
 			// AppID
 
-			"ibm_appid_application":        resourceIBMAppIDApplication(),
-			"ibm_appid_application_scopes": resourceIBMAppIDApplicationScopes(),
-			"ibm_appid_application_roles":  resourceIBMAppIDApplicationRoles(),
-			"ibm_appid_token_config":       resourceIBMAppIDTokenConfig(),
-			"ibm_appid_redirect_urls":      resourceIBMAppIDRedirectURLs(),
-			"ibm_appid_role":               resourceIBMAppIDRole(),
+			"ibm_appid_application":         resourceIBMAppIDApplication(),
+			"ibm_appid_application_scopes":  resourceIBMAppIDApplicationScopes(),
+			"ibm_appid_application_roles":   resourceIBMAppIDApplicationRoles(),
+			"ibm_appid_idp_cloud_directory": resourceIBMAppIDIDPCloudDirectory(),
```

Comment:
will be adding more resources, like `_idp_facebook`, `_idp_saml` thus chose this naming format